kafka/producer/main.go

```diff
@@ -197,21 +190,29 @@ func generateRandomNum(span int, targetRate int) int {
 	return rand.Intn((targetRate+span)-min) + min + 1
 }
 
-func loopPrint() {
-	for {
-		time.Sleep(5 * time.Second)
-		log.Printf("total Msg is: %d", totalMsg)
-		log.Printf("average burst rate is: %d", int(totalMsg)/totalBurst)
-		log.Printf("average rest duration is: %d", totalRestDur/totalBurst)
-		log.Printf("average send duration is: %d", totalSendDur/totalBurst)
+func getEnv(envName string) string {
+	value := os.Getenv(envName)
+	if value == "" {
+		fmt.Println("Environment variable " + envName + " is missing")
```

Comment:
`fmt.Println("Environment variable %q is missing", envName)` will quote the envName automatically.  This will help with issues where there is accidential ' ' before or at the end of the env.